framework/integration/init.go

```diff
@@ -0,0 +1,101 @@
+// +build k8srequired
+
+package integration
+
+import (
+	"github.com/giantswarm/e2e-harness/pkg/harness"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/framework"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	namespace = "finalizer-integration-test"
+)
+
+var (
+	err error
+
+	k8sClient     kubernetes.Interface
+	k8sRESTClient rest.Interface
+)
+
+func init() {
+	k8sClient, err = newK8sClient()
+	if err != nil {
+		panic(err)
+	}
+	k8sRESTClient, err = newK8sRESTClient()
+}
+
+func newK8sClient() (kubernetes.Interface, error) {
+	config, err := clientcmd.BuildConfigFromFlags("", harness.DefaultKubeConfig)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	k8sClient, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	return k8sClient, nil
+}
+
+func newK8sRESTClient() (rest.Interface, error) {
+	config, err := clientcmd.BuildConfigFromFlags("", harness.DefaultKubeConfig)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	k8sRESTClient, err := rest.RESTClientFor(config)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	return k8sRESTClient, nil
+}
+
+func newFramework(name string) (*framework.Framework, error) {
+	logger, err := micrologger.New(micrologger.DefaultConfig())
+	if err != nil {
+		return nil, err
+	}
+	f := framework.New()
```

Comment:
Is there a way to just specify the things needed for my tests? Or should I just initialize the full framework here?